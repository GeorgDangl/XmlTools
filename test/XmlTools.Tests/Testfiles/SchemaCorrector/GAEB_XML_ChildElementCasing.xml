<?xml version="1.0" encoding="UTF-8"?>
<GAEB>
  <gaebinfo></gaebinfo>
  <PrjInfo></PrjInfo>
  <Award></Award>
</GAEB>

