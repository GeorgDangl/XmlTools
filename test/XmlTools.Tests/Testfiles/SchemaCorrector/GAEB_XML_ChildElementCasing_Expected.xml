<?xml version="1.0" encoding="UTF-8"?>
<GAEB>
  <GAEBInfo></GAEBInfo>
  <PrjInfo></PrjInfo>
  <Award></Award>
</GAEB>

